packages/js/data/changelog/55621-fix-product-editor-ts-errors-4

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+`@woocommerce/data`: Enable TS check.
```

Comment:
We are also still making a lot of type changes here as part of the crud data store, should we also mention this?
Or are you skipping that as this would be covered by your other PR and the changelog that mentioned it already?

Reply:
Good point! I updated with dedb4101f8d0aedd89dacdcd505bf5e4ddc7bd95